support/loggers/loggers.go

```diff
@@ -0,0 +1,53 @@
+package loggers // I didn't call the packacke "logging" because the go-loggeing framework's package is "logging" so we'll need to be different when we switch
+
+import (
+	"log"
+)
+
+type Logger interface {
+	// basic messages, appends a newline (\n) after each entry
+	Info(msg string)
+
+	// basic messages, can be custom formatted, similar to fmt.Printf. User needs to add a \n if they want a newline after the log entry
+	Infof(msg string, args ...interface{})
+
+	// error messages, indicates to the logger that these messages can be handled differently (different color, special format, email alerts, etc.). The type of logger will determine what to do with these messages. The logger should NOT panic on these messages. Appends a newline (\n) after each entry.
+	Error(msg string)
+
+	// error messages, indicates to the logger that these messages can be handled differently (different color, special format, email alerts, etc.). The type of logger will determine what to do with these messages. The logger should NOT panic on these messages. User needs to add a \n if they want a newline after the log entry.
+	Errorf(msg string, args ...interface{})
+
+	// added Fatal and Fatalf because trade.go (and elsewhere) use log.Fatal
```

Comment:
I'd rather that we remove `Fatal` and `Fatalf` as methods on the interface for now -- we can use `os.exit()` in the places where we call `log.Fatal` (very few, only at the top level). This will reduce the burden on the log implementations to have to implement these additional two methods which are essentially just an `Error` followed by an `os.exit()`

we should add a method here to help with that:
```golang
// Fatal is a convenience method that can be used with any Logger to log a fatal error
func Fatal(l Logger, e error) {
    l.Info("")
    l.Error(e)
    os.exit(1)
}
```

this will be used like so:
```golang
// some code that gives us an e that we want to log as a fatal error
logger.Fatal(l, e)
```